src/adaptation/Adaptation.jl

```diff
@@ -30,7 +30,7 @@ function getϵ(ca::AbstractCompositeAdaptor)
     return getϵ(ca.ssa)
 end
 
-include("threephase.jl")
+include("stannuts.jl")
```

Comment:
This 
Consider rename `stannuts.jl` to `stan_adaption.jl`, since `stannuts.jl` contains only the windowed adaption implementation.